.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
+/lib/
```

Comment:
Dit werkt wel alleen als je de package dan ook submit als package (wat we tot nu toe niet deden)

Reply:
juist... hmmm